samples/dialect/BUILD.bazel

```diff
@@ -0,0 +1,25 @@
+load("@rules_sqldelight//:sqldelight.bzl", "sqldelight_codegen")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "define_kt_toolchain", "kt_jvm_library")
+
+kt_jvm_library(
+    name = "library",
+    srcs = glob(["src/main/java/**/*.java"]) + [":generated_sources"],
+    deps = [
+        "@maven//com/squareup/sqldelight:jdbc-driver",
```

Comment:
Does this automatically fetch the right JDBC driver? That's pretty handy.

Reply:
It's not fetching the JDBC driver itself - for compilation, this is the sqldelight driver for JDBC. Lets them use JDBC under the cover. You'd still have to have the right JDBC driver proper in the runtime classpath, I think.